R/jjcsformats.R

```diff
@@ -0,0 +1,230 @@
+#' Title: Function factory for xx style formatting
+#'
+#' @description A function factory to generate formatting functions for value
+#' formatting that support the xx style format and control the rounding method
+#' @importFrom tidytlg roundSAS
+#' @param roundmethod (`string`)\cr choice of rounding methods. Options are:
+#'   * `SAS`: the underlying rounding method is `tidytlg::roundSAS`, where \cr
+#'   roundSAS comes from this Stack Overflow post https://stackoverflow.com/questions/12688717/round-up-from-5
+#'   * `R`: the underlying rounding method is `round`
+#'
+#' @return `jjcs_format_xx_fct()` format function that can be used in rtables formatting calls
```

Comment:
why jjcs? I think these functions may well be of general use, so probably better to have a user-friendly name for these format functions?

Reply:
jjcs was coming from our internal project, I agree that we should come up with a better name for this, if you would have any suggestions, these would be more than welcome